deliver/lib/deliver/upload_metadata.rb

```diff
@@ -55,6 +55,8 @@ class UploadMetadata
     # Directory name it contains review information
     REVIEW_INFORMATION_DIR = "review_information"
 
+    # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
Do this value make sense?

https://github.com/giginet/fastlane/blob/phased-release/.rubocop.yml#L172